mozilla/blurts-server
# MNTOR-2120 - add Sentry logging for publisher, and skip pubsub for Heroku

# References: 
Jira: MNTOR-2120



# Description

- The `api/v1/hibp/notify` route is a publisher, and if it drops messages they will be lost. Use Sentry to log these, since the messages are quite minimal.
- Heroku has no access to pubsub, detect this environment via `APP_ENV` and skip pubsub in that env.

# How to test

- Run locally w/o access to PubSub emulator and ensure that Sentry logging happens.
- Ensure that Heroku can process breach alerts.

# Checklist (Definition of Done)
